### ISSUE TITLE
My program compiled with Nuitka is blocked by all antivirus

### ISSUE THREAD

**Issue by Ambro86** [Nuitka/Nuitka#2163]
Hello everyone, and first of all thank you for developing Nuitka, which allows us to easily distribute our programs. I have created a program for the blind, a builder, which allows you to create maps in an accessible way, using sounds, building them with different pavements, with doors, being able to build buildings, with stairs, different floors, etc.
My program uses pygame, a library for speaking messages, which is called accessible_output2, and other common libraries including zipfile, json, etc.
By compiling it with nuitka, most antiviruses block it, saying it's a virus, and even a high-level one.
Windows defender classifies it like this: Trojan:Win32/Wacatac.H!ml
While I'm at it, I tell you that before I compiled the game as a single file, with the onefile option, but I noticed that the antivirus blocked it instantly, so I'm compiling it with several scattered files.
Can you help me if there is a way to be less attacked by antivirus?
This is the command I use to compile the game:
python -m nuitka --standalone --windows-disable-console --include-data-dir="C:\Users\xxx\Documents\distribution"= builder.py
The distribution folder I include contains the game sounds.
This is link of my project: https://ambro86.itch.io/map-builder-for-blind
I leave you my specifications
Nuitka 1.5.5
Commercial: None
Python: 3.9.10 (tags/v3.9.10:f2f3f53, Jan 17 2022, 15:14:21) [MSC v.1929 64 bit (AMD64)]
Flavor: CPython Official
Executable: C:\Users\xxx\AppData\Local\Programs\Python\Python39\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Version C compiler: ~\AppData\Local\Nuitka\Nuitka\Cache\downloads\gcc\x86_64\11.3.0-14.0.3-10.0.0-msvcrt-r3\mingw64\bin\gcc.exe (gcc).

**Comment by kayhayen** [comment]
The 1.5.6 release is going to bring the number of AV that flags hello world down quite a bit, but not to zero. I am working with AV vendors that still flag it. Hello world was flagged by 8 on Virustotal. With 1.5.5 I got it to 7, and now it's going to be 4. I am talking with them, to reduce it to zero eventually.

In Nuitka commercial documentation, there is a full set of instructions how to overcome this for your project. Without your source code, this lacks an important piece of input. Surely I am not going to even look at your binaries. So without a reproducer, I will have to treat this as invalid unless you provide the source.

Assistance technologies will be able to do things that malware typically does. I think AV will have a hard time with your kind of software anyway. I don't really see how screen capture and potentially unattended input will be allowed for if not following the instructions for professional users.

**Comment by Ambro86** [comment]
Thanks for your response. If you deem it useful, I can privately
provide you with my Builder's code. Commercial Nuitka's directions I
guess are only valid if you activate a commercial license. I am a
small developer, also this project is freeware.
Thanks again.

2023-04-12 7:42 GMT+02:00, Kay Hayen ***@***.***>:

**Comment by sante85** [comment]
Dear, i have a question? you not have error as modulenotfound? if you import external modules?

**Comment by sante85** [comment]
can make  trivial example? thanks

**Comment by Ambro86** [comment]
Sorry, I'm not sure I understand what you're saying. Want an example of using accessibility? If so, here is some of the code
import accessible_output2.outputs.auto
o = accessible_output2.outputs.auto.Auto()
def speak(items):
    o.speak(f"{items}")
    At this point you can use the method I created to have the desired messages spoken, for example
    Speak("This is a test!")

**Comment by dlsofree** [comment]
add --clang

**Comment by Ambro86** [comment]
@dlsofree  Thanks a lot, I'll try!
@kayhayen Let me know if you find it useful to view my code.

**Comment by kayhayen** [comment]
For me, a freeware should have its source code available to be considered free. That term is from a different age, things are now different. If you produce a minimal example that shows how accessibility code, that I can use without special equipment, flags to AV, I am going to benefit from that. I am not interested in supporting closed software free of charge.

Please open a new issue, if you have such an example.
